server/app.js

```diff
@@ -138,6 +139,59 @@ app.get('/migrate', async (req, res, next)=>{
 	return next();
 });
 
+//Account Information page
+app.get('/account', async (req, res, next)=>{
+	let text = dedent`
+	# Account Details
+	
+	Not logged in.`;
+
+	let auth;
+	let files;
+	if(req.account) {
+		if(req.account.googleId) {
+			try {
+				auth = await GoogleActions.authCheck(req.account, res);
+			} catch (e) {
+				auth = undefined;
+				console.log('Google auth check failed!');
+				console.log(e);
+			}
+			if(auth.credentials.access_token) {
+				try {
+					files = await GoogleActions.listGoogleBrews(auth);
+				} catch (e) {
+					files = undefined;
+					console.log('List Google files failed!');
+					console.log(e);
+				}
+			}
+		}
+
+		text = dedent`
+	# Account Details
+	:
+	{{wide,centered,text-align:center,border-color:#58180D,border-width:2px,border-style:solid,border-radius:10px,padding:10px
+	### Info
+	:
+	|||
+	|:-|:-|
+	|Username|${req.account.username}|
+	|Logged in at|${req.account.issued}|
+	|Linked to Google|${req.account.googleId ? 'YES' : 'NO'}|
+	${req.account.googleId ? `|Google auth check: ${auth.credentials.access_token ? '|YES|' : '|NO|'}` : ''}
+	${req.account.googleId ? `|Google file list count: ${files ? `|${files.length}|` : '|NO|'}` : ''}
+	}}`;
+	}

```

Comment:
I'm not certain how I feel about this templating vs writing a React template that accepts props. Was there a reason for choosing this strategy over React+route props?

Reply:
As WIP, finding out what data was available to be presented, and what it represented in human terms was my first concern; presentation was a secondary concern.
A proper template should be created that is visually distinct from normal Homebrewery brews that ideally can be re-used for future UI pages - as `basePages/uiPage/uiPage.jsx`, probably with an associated `basePages/uiPage/uiPage.less`.

We should create a separate PR for creating a reusable UI page template that accepts props, which we can then populate with this data. This Draft PR was mostly a quick proof of concept, and once the scope widened, it stalled out.

Reply:
Oh you commented on that in the gitter chat! Okay sounds good!

Reply:
Oh hey, I already did that, creating #2049.